project/legislation/openstates/test_pyopenstates.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bn/env python
```

Comment:
We don't need to keep these unit tests in our repository, as they are maintained in the upstream project.